[2024-07-31T15:52:05.835+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Pipeline_ETL_dataRaw_to_dataClean.FetchFromPostgresql scheduled__2024-07-30T06:30:00+00:00 [queued]>
[2024-07-31T15:52:05.853+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Pipeline_ETL_dataRaw_to_dataClean.FetchFromPostgresql scheduled__2024-07-30T06:30:00+00:00 [queued]>
[2024-07-31T15:52:05.856+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2024-07-31T15:52:05.858+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 7
[2024-07-31T15:52:05.859+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2024-07-31T15:52:05.921+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): FetchFromPostgresql> on 2024-07-30 06:30:00+00:00
[2024-07-31T15:52:05.945+0000] {standard_task_runner.py:52} INFO - Started process 663 to run task
[2024-07-31T15:52:05.978+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Pipeline_ETL_dataRaw_to_dataClean', 'FetchFromPostgresql', 'scheduled__2024-07-30T06:30:00+00:00', '--job-id', '49', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_ETL_dataRaw_to_dataClean.py', '--cfg-path', '/tmp/tmpswqpoonm', '--error-file', '/tmp/tmpia1fq4bv']
[2024-07-31T15:52:05.982+0000] {standard_task_runner.py:80} INFO - Job 49: Subtask FetchFromPostgresql
[2024-07-31T15:52:06.246+0000] {task_command.py:371} INFO - Running <TaskInstance: Pipeline_ETL_dataRaw_to_dataClean.FetchFromPostgresql scheduled__2024-07-30T06:30:00+00:00 [running]> on host 44d2c61df54a
[2024-07-31T15:52:06.482+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yudis_aditya3
AIRFLOW_CTX_DAG_ID=Pipeline_ETL_dataRaw_to_dataClean
AIRFLOW_CTX_TASK_ID=FetchFromPostgresql
AIRFLOW_CTX_EXECUTION_DATE=2024-07-30T06:30:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-07-30T06:30:00+00:00
[2024-07-31T15:52:06.751+0000] {python.py:173} INFO - Done. Returned value was: None
[2024-07-31T15:52:06.786+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=Pipeline_ETL_dataRaw_to_dataClean, task_id=FetchFromPostgresql, execution_date=20240730T063000, start_date=20240731T155205, end_date=20240731T155206
[2024-07-31T15:52:06.859+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-07-31T15:52:07.686+0000] {local_task_job.py:279} INFO - 0 downstream tasks scheduled from follow-on schedule check
